src/main/java/org/project_kessel/relations/example/Caller.java

```diff
@@ -241,4 +242,49 @@ public static void lookupSubjectsExample() {
         System.out.println("Collected reactive non-blocking responses: " + responses2);
     }
 
+    public static void lookupResourcesExample() {
+        var url = "localhost:9000";
+
+        /* Make a secure connection with grpc TLS this time */
```

Comment:
I know it's copied from above, but I wonder if we should adjust or remove the comment. It's saying it's creating a secure connection and then creating an insecure one.

Reply:
I agree, I can remove it